spec/common_scripting_layer/pie_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+describe 'Pie' do
+  before do
+    @datasets = [
+      [:China, [25]],
+      [:India, [80]],
+      [:Nepal, [22]],
+      [:Germany, [95]],
+      [:Australia, [90]]
+    ]
+  end
+
+  it 'Sets up a basic reference image for histogram with random numbers' do
+    plot = Rubyplot::Pie.new
+    plot.title = 'Visual Pie Graph Test'
+    @datasets.each do |data|
+      plot.data(data[0], data[1])
+    end
+    plot.write('spec/reference_images/pie_test_1.png')
+
+    expect(compare_with_reference?('pie.png', 'pie_test_1.png', 10)).to eq(true)
+  end
+
+  it 'test pie graph nearly equal' do
+    plot = Rubyplot::Pie.new
+    plot.title = 'Pie Graph Nearly Equal'
+
+    plot.data(:Arafat, [41])
+    plot.data(:Pranav, [42])
+    #    plot.data(:Grouch, [40])
+    #    plot.data(:Snuffleupagus, [43])
+
+    plot.write('test/output/pie_nearly_equal.png')
+  end
+
+  it 'pie graph equal' do
```

Comment:
So, these are really supposed to be written in grammatical English. That's the whole point of this weird `it` notation. These should be formulated as sentences, and if you can't formulate them that way, they may not be good specs (and certainly aren't clear to readers).

`it 'should produce a pie graph with equal proportions' do`, for example.

Reply:
Okay. Will change all of these don't worry about it.